lib/router.ex

```diff
@@ -0,0 +1,22 @@
+defmodule HTTPServer.Router do
+  alias HTTPServer.Response
+  alias HTTPServer.Request
+
+  def route(%Request{method: "POST", path: "/echo_body"} = req) do
+    Response.build_response(req, 200)
+  end
+
+  def route(%Request{method: "GET", path: "/simple_get"} = req) do
+    Response.build_response(req, 200)
+  end
```

Comment:
I think there is a potential bug here. If your `req` has a body, it will be echoed because of how `build_response` is defined on the `Response`